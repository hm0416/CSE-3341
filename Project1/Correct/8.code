program
	int x,y,z;
	ref a	,b	,c	;
begin
	input x;
	output 4;
	a	=	new;
end
