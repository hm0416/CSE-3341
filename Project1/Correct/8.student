['s', 'r']
['s', 'r']
0
0
