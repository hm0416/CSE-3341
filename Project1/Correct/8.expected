INT
ID[x]
COMMA
ID[y]
COMMA
ID[z]
SEMICOLON
